ui/src/app/workflows/components/workflows-row/workflows-row.scss

```diff
@@ -0,0 +1,8 @@
+@import 'node_modules/argo-ui/src/styles/config';
+
+.wf-rows-name {
+    white-space: pre-line;
+    line-height: 1.5em;
+    display: inline-block;
+    vertical-align: middle;
+}
```

Comment:
Can't remember which one of these ignores the `overflow: hidden` from, I think, [here](https://github.com/argoproj/argo-workflows/blob/e7354da46258d742393af8d5c99ef7266b433661/ui/src/app/workflows/components/workflows-list/workflows-list.scss#L180-L182).  Figured we'd want the descriptions to be able to show all content.  Or do we? 🤔